src/test/cls_sdk/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_executable(ceph_test_cls_sdk
+  test_cls_sdk.cc
+  )
+set_target_properties(ceph_test_cls_sdk PROPERTIES COMPILE_FLAGS
+  ${UNITTEST_CXX_FLAGS})
+target_link_libraries(ceph_test_cls_sdk
+  librados
+  global
+  ${EXTRALIBS}
+  ${BLKID_LIBRARIES}
+  ${CMAKE_DL_LIBS}
+  radostest
+  ${UNITTEST_LIBS}
+  )
+install(TARGETS
```

Comment:
I mostly followed this https://github.com/ceph/ceph/blob/master/src/test/cls_hello/CMakeLists.txt#L15. But I can always incorporate your suggestion.